recipes/xcp/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.22.0" %}
+
+package:
+  name: rustic
```

Comment:
Why is the package name rustic?

Reply:
Ohh, that's why. Sorry I missed something so obvious. I'll create a new PR